### DISCUSSION TITLE
Best ESP32 for all entities

### DISCUSSION THREAD

**Discussion by MBRetrofit** [syssi/esphome-smg-ii#83] [General]
What's the minimum CPU/RAM/ROM for ESP32 for enabling the full feature for this smg-ii integration?

**Comment by syssi** [comment]
Any dual core ESP32 should do the job. Please avoid single core models.

**Comment by MBRetrofit** [comment]
Works great, thank you!

Is there any registry for Firmware Version?

**Reply by syssi** [reply]
I don't understand you question. Which firmware are you talking about?

**Reply by MBRetrofit** [reply]
Invertor Firmware, I have Easun SMG II

**Reply by syssi** [reply]
I'm not aware of any good knowledge source about the inverter. If you find more details about the inverter, I will be happy to add them to the README.

**Reply by MBRetrofit** [reply]
https://github.com/gregoriusin/easun-smg-ii-ha-config

Anyhow, my SMG II works great with your repository.

About the Firmware, somewhere in the Display menu on the Inverter there is information about displaying the Firmware Version of the Invertor. That's why I asked if there is any info about how can the Firmware version can be displayed over the ESP Home
